test/e2e_test.dart

```diff
@@ -56,17 +47,24 @@ void main() {
 
       group('create', () {
         test('-t dart_pkg', () async {
-          final directory = Directory(path.join('.tmp', 'very_good_dart'));
+          final directory = Directory.systemTemp.createTempSync();
```

Comment:
shouldn't we still be deleting the temp directories after each run?

Reply:
Afaik temp dirs will eventually be cleaned up once the process attached is done or when the system does its garbage sweep.

I am fine with adding it, but I didn't feel it was that necessary